sct_tutorial/SCT_Tutorial_01_Segmentation.ipynb

```diff
@@ -0,0 +1,445 @@
+{
```

Comment:
## Cons

* Difficult to maintain
    * Jupyter notebooks are tricky to version-track using Git
    * This would add a fouth source of truth for the tutorials (in addition to the `.rst` pages, the `batch_single_subject.sh` script, and the SCT Course slides). Making a change to the tutorials would now require keeping even more things in sync with one another.
 * Might make using SCT more complicated for users
    * Users would now have extra options for what the "proper" way to use SCT is (too many choices --> confusion)
    * Documentation might be more challenging to navigate for users (Which page/section do we put these notebooks in? Will users know to go to the tutorial pages, or the notebooks?)
    * They will likely have to learn CLI skills anyway if they want to fully take advantage of SCT (e.g. batch scripting), so providing this resource might be delaying the inevitable for them